src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2584,12 +2614,23 @@ void Player_StartChangingHeldItem(Player* this, PlayState* play) {
 }
 
 void Player_UpdateItems(Player* this, PlayState* play) {
+#if OOT_VERSION < NTSC_1_1
+    if ((this->actor.category == ACTORCAT_PLAYER) &&
+        ((this->heldItemAction == this->itemAction) || (this->stateFlags1 & PLAYER_STATE1_22)) &&
+        (gSaveContext.save.info.playerData.health != 0) && (play->csCtx.state == CS_STATE_IDLE) &&
+        (this->csAction == PLAYER_CSACTION_NONE) && (play->shootingGalleryStatus == 0) &&
+        (play->activeCamId == CAM_ID_MAIN) && (play->transitionTrigger != TRANS_TRIGGER_START) &&
+        (gSaveContext.timerState != TIMER_STATE_STOP))
+#else
+    // !(this->stateFlags1 & PLAYER_STATE1_START_CHANGING_HELD_ITEM) added
     if ((this->actor.category == ACTORCAT_PLAYER) && !(this->stateFlags1 & PLAYER_STATE1_START_CHANGING_HELD_ITEM) &&
         ((this->heldItemAction == this->itemAction) || (this->stateFlags1 & PLAYER_STATE1_22)) &&
         (gSaveContext.save.info.playerData.health != 0) && (play->csCtx.state == CS_STATE_IDLE) &&
         (this->csAction == PLAYER_CSACTION_NONE) && (play->shootingGalleryStatus == 0) &&
         (play->activeCamId == CAM_ID_MAIN) && (play->transitionTrigger != TRANS_TRIGGER_START) &&
-        (gSaveContext.timerState != TIMER_STATE_STOP)) {
+        (gSaveContext.timerState != TIMER_STATE_STOP))
+#endif
```

Comment:
yeah I still think it'd be better to #if the specific expressions rather than the whole conditional

Reply:
Alright, I changed the 2 egregious case in this file as a trial run